.mailmap

```diff
@@ -1,32 +1,151 @@
+Misty Stanley-Jones <misty@apache.org>
+Misty Stanley-Jones <misty@apache.org> <misty@docker.com>
+Nikita Tarasov <nikita@mygento.ru>
+Nikita Tarasov <nikita@mygento.ru> <luckyraul@users.noreply.github.com>
+Oleg Bulatov <oleg@bulatov.me>
+Oleg Bulatov <oleg@bulatov.me> <obulatov@redhat.com>
+Olivier Gambier <olivier@docker.com>
+Olivier Gambier <olivier@docker.com> <dmp42@users.noreply.github.com>
+Omer Cohen <git@omer.io>
+Omer Cohen <git@omer.io> <git@omerc.net>
+Per Lundberg <perlun@gmail.com>
+Per Lundberg <perlun@gmail.com> <per.lundberg@ecraft.com>
+Peter Dave Hello <hsu@peterdavehello.org>
+Peter Dave Hello <hsu@peterdavehello.org> <PeterDaveHello@users.noreply.github.com>
+Phil Estes <estesp@gmail.com>
+Phil Estes <estesp@gmail.com> <estesp@amazon.com>
+Phil Estes <estesp@gmail.com> <estesp@linux.vnet.ibm.com>
+Richard Scothern <richard.scothern@gmail.com>
+Richard Scothern <richard.scothern@gmail.com> <richard.scothern@docker.com>
+Rober Morales-Chaparro <rober.morales@rstor.io>
+Rober Morales-Chaparro <rober.morales@rstor.io> <rober@rstor.io>
+Sharif Nassar <sharif@mrwacky.com>
+Sharif Nassar <sharif@mrwacky.com> <mrwacky42@users.noreply.github.com>
+Sebastiaan van Stijn <github@gone.nl>
+Sebastiaan van Stijn <github@gone.nl> <moby@example.com>
+Sebastiaan van Stijn <github@gone.nl> <sebastiaan@ws-key-sebas3.dpi1.dpi>
+Sebastiaan van Stijn <github@gone.nl> <thaJeztah@users.noreply.github.com>
+Joffrey F <joffrey@docker.com>
+Joffrey F <joffrey@docker.com> <f.joffrey@gmail.com>
+Joffrey F <joffrey@docker.com> <joffrey@dotcloud.com>
+Solomon Hykes <solomon@docker.com>
+Solomon Hykes <solomon@docker.com> <s@docker.com>
+Solomon Hykes <solomon@docker.com> <solomon.hykes@dotcloud.com>
+Solomon Hykes <solomon@docker.com> <solomon@dotcloud.com>
+Stephen Day <stevvooe@gmail.com>
+Stephen Day <stevvooe@gmail.com> <stephen.day@docker.com>
+Stephen Day <stevvooe@gmail.com> <stevvooe@users.noreply.github.com>
+Sven Dowideit <SvenDowideit@home.org.au>
+Sven Dowideit <SvenDowideit@home.org.au> <SvenDowideit@users.noreply.github.com>
+Tibor Vass <teabee89@gmail.com>
+Tibor Vass <teabee89@gmail.com> <tibor@docker.com>
+Tibor Vass <teabee89@gmail.com> <tiborvass@users.noreply.github.com>
+Cristian Staretu <cristian.staretu@gmail.com>
+Cristian Staretu <cristian.staretu@gmail.com> <unclejack@users.noreply.github.com>
+Cristian Staretu <cristian.staretu@gmail.com> <unclejacksons@gmail.com>
+Victor Vieux <victor.vieux@docker.com>
+Victor Vieux <victor.vieux@docker.com> <dev@vvieux.com>
+Victor Vieux <victor.vieux@docker.com> <victor.vieux@dotcloud.com>
+Victor Vieux <victor.vieux@docker.com> <victor@docker.com>
+Victor Vieux <victor.vieux@docker.com> <victor@dotcloud.com>
+Victor Vieux <victor.vieux@docker.com> <victorvieux@gmail.com>
+Victor Vieux <victor.vieux@docker.com> <vieux@docker.com>
+Vincent Batts <vbatts@redhat.com>
+Vincent Batts <vbatts@redhat.com> <vbatts@hashbangbash.com>
+Vincent Demeester <vincent.demeester@docker.com>
```

Comment:
Do you have any suggestions for how to do this? I believe it should stay as is because the contribution as recorded in the git commit tree was made with the email address as stated above. Otherwise we're down the world of _INTENSE_ pain of querying GH API.

Reply:
I wouldn't bother _too much_, but there's a bunch in this one that I'm aware of (former colleagues and such). Removing duplicates may be more relevant to get a (somewhat) accurate list of authors.

Reply:
Yeah, I agree RE: removing dupes. That's a solvable problem, searching for up-to-date email addresses I would not bother with.

Reply:
Yup. I might do a follow-up to set a reasonable baseline (fixing the ones I'm aware of), but after that we can limit to / focus on just removing duplicates (which are easier to spot).

FWIW; CNCF also maintains a database for this purpose, where they try to centrally map aliases etc for their stats (it's being replaced with some other internal tool though); https://github.com/cncf/devstats